### ISSUE TITLE
After upgrading to v3.4.0, I got an error running Ember CLI: An addon must define a `name` property.

### ISSUE THREAD

**Issue by varoot** [netguru/eslint-plugin-ember#73]
This is what I got, even if I just run `ember -v`:

```
An addon must define a `name` property.
An error occured in the constructor for eslint-plugin-ember at /Users/username/repo/node_modules/eslint-plugin-ember
```

**Comment by varoot** [comment]
I think because of the latest release, it registers this as an Ember addon, and therefore "name" key is required in `index.js`

**Comment by michalsnik** [comment]
Fixed in `v3.4.1`
